internal/report.go

```diff
@@ -404,6 +404,12 @@ func buildTableReportBody(conv *Conv, srcTable string, issues map[string][]Schem
 				case InterleavedAddColumn:
 					str := fmt.Sprintf(" %s add %s as a primary key in table %s", IssueDB[i].Brief, srcCol, spSchema.Name)
 
+					if !contains(l, str) {
+						l = append(l, str)
+					}
+				case InterleavedRenameColumn:
```

Comment:
In case of multiple foreign keys , this statement would be unclear. Add the info about which foreign key's refer column is being referred to here.